microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -22,149 +22,158 @@ ms.date: 05/04/2021
+The code changes the background color of the webpage to red, adds a new line of text at the bottom of the webpage, and logs a message to the **Console** tool.
 
-![The webpage after running the Snippet](./snippets-images/javascript-sources-snippets-split-screen-after.png)
+When you run a snippet on a webpage, the snippet's source code is added to the current webpage. For more information about changing the existing code of a webpage, instead of adding new code, see [Override webpage resources with local copies (Overrides tab)](./overrides.md).
+
+**Note:** The security settings of most webpages block from loading other scripts in snippets. For this reason, you must include all your code in one file.
 
 
 <!-- ====================================================================== -->
 ## Open the Snippets tab
 
-The **Snippets** tab, in the **Navigator** pane on the left, lists your Snippets.  When you want to edit a Snippet, you need to open it from the **Snippets** tab:
+The **Snippets** tab is located in the **Navigator** pane, on the left of the **Sources** tool.
 
-![The Snippets tab](./snippets-images/javascript-sources-snippets-pane.png)
+To open the **Snippets** tab:
 
-### Open the Snippets tab with a mouse
+1. To open DevTools, right-click the webpage, and then select **Inspect**. Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS). DevTools opens.
 
-1. Select the **Sources** tab.  The **Sources** tool appears:
+1. In DevTools, on the Activity Bar, select the **Sources** tab. If that tab isn't visible, click the **More tools** (![More tools icon](./snippets-images/more-tools-icon.png)) button.
 
-   ![The Sources tool with the Page tab open on the left](./snippets-images/javascript-sources-page-pane.png)
+   ![The Sources tool with the Page tab open on the left](./snippets-images/sources-tool-page-pane.png)
 
 1. In the **Navigator** pane (on the left), select the **Snippets** tab.  To access the **Snippets** option, you might need to click the **More tabs** (![More tabs](./snippets-images/more-tabs-icon.png)) button.
 
-### Open the Snippets tab with the Command Menu
+#### Open the Snippets tab with the Command Menu
+
+You can also open the **Snippets** tab with the Command Menu:
 
 1. Select anything in DevTools, so that DevTools has focus.
 
 1. Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS) to open the Command Menu.
 
-1. Type **Snippets**, select **Show Snippets**, and then press **Enter** to run the command:
+1. Start typing "Snippets", select **Show Snippets**, and then press **Enter** to run the command:
 
-   ![The Show Snippets command](./snippets-images/javascript-search-show-snippets.png)
+   ![The Show Snippets command](./snippets-images/command-menu.png)
 
 
 <!-- ====================================================================== -->
-## Create Snippets
+## Create new snippets
```

Comment:
```suggestion
## Create a new snippet
```
the concrete task is creating _a_ snippet.  User creates 1 at a time.  generally, singular is clearer than plural; less ambiguous.